app/retail/templates/shared/footer_scripts.html

```diff
@@ -28,6 +28,7 @@
     <script src="{% static "v2/js/jquery.validate.min.js" %}"></script>
     <script src="{% static "v2/js/jsrender.js" %}"></script>
     <script src="{% static "v2/js/base.js" %}"></script>
+    <script src="{% static "v2/js/purify.min.js" %}"></script>
```

Comment:
@kziemianek `footer_scripts` gets imported in every page, it looks like we are using it everywhere

make sense to remove it other places rather than importing it twice  ? 

https://github.com/gitcoinco/web/pull/1392/files#diff-61e7807f7dc290de5b5feabd97d44778L112

Reply:
I'm going to left it as it is (analogously with shared.js, in order to solve submitted bug) ok?
further refactor in another PR k?

Reply:
Ah we've got ton of tech debt piling up !
Could you update it within this PR ? 
It's just removing it from the other files
and leaving it as is in `footer_scripts`

Pretty please to make the maintainers life a lil easier  ? :P

Reply:
i'll left purify import only in footer_scripts.html